src/chapters/exceptions/exercises.rst

```diff
@@ -0,0 +1,29 @@
+Exercises
+=========
+
+To get started, fork and clone this `repo <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn9exceptions>`_ from Github. 
+
+Divide By Zero!
+---------------
+
+Complete a function called ``Divide()`` in ``Program.cs``.
+``Divide()`` takes in two parameters: ``x`` and ``y``.
+
+Your function should return the result of ``x/y``.
+
+However, if ``y`` is zero you should throw an appropriate error message.
+Try to use an ``ArgumentOutOfRangeException`` and put your ``try/catch`` block in ``Main()`` to test out your error-handling skills!
```

Comment:
This goes for both of these methods - I think it's a good idea to say explicitly that if an exception is caught, then they should print out the error message

Reply:
Done!